### PR TITLE
Add overview of the procedure of contributing software to the EESSI software stack

### PR DESCRIPTION
**Pull request by xinan1911**

### CODE REVIEW

**Comment by xinan1911** [comment]
@boegel Could you take a look at this?

**Review by casparvl — DISMISSED** [review]
In the latest sync meeting, @boegel mentioned it would be good to have the source files for creating the flowchart as well, as it will allow for future adaptations if needed.

I think this was created in Powerpoint, right? Could you include the Powerpoint?

**Comment by xinan1911** [comment]
> In the latest sync meeting, @boegel mentioned it would be good to have the source files for creating the flowchart as well, as it will allow for future adaptations if needed.
> 
> I think this was created in Powerpoint, right? Could you include the Powerpoint?

Source file powerpoint added.

File: docs/adding_software/overview.md [hunk 1]
```diff
@@ -1,4 +1,24 @@
-# Adding software to EESSI
+# Overview of adding software to EESSI
+
+We welcome contributions to EESSI software stack. This page shows an overview of the procedures of contributing to the EESSI repository.
```
Comment (trz42) [inline docs/adding_software/overview.md:3]:
```suggestion
We welcome contributions to the EESSI software stack. This page shows the procedure and provides links to the contribution policy and the technical details of making a contribution. 
```

File: docs/adding_software/overview.md [hunk 2]
```diff
@@ -1,4 +1,24 @@
...
+
+We welcome contributions to EESSI software stack. This page shows an overview of the procedures of contributing to the EESSI repository. 
+
+## Contribute a software to EESSI software stack
+
+<p align="center">
+  <img width="50%" src="../img/contribute_software_workflow.png" alt="EESSI contribute software">
+</p>
+
+## Contributing a ReFrame test to EESSI test suite
```
Comment (trz42) [inline docs/adding_software/overview.md:11]:
```suggestion
## Contributing a ReFrame test to the EESSI test suite
```

File: docs/adding_software/overview.md [hunk 3]
```diff
@@ -1,4 +1,24 @@
...
+
+Ideally, contributor makes a ReFrame test for the software he plans to contribute.
```
Comment (trz42) [inline docs/adding_software/overview.md:13]:
```suggestion
Ideally, a contributor prepares a ReFrame test for the software to be added to the EESSI software stack. 
```

File: docs/adding_software/overview.md (hunk 1 above)
Comment (xinan1911) [reply docs/adding_software/overview.md:3]: Thanks for the suggestions. They all look good.

**Pull request merged**